template/libtofs.jinja

```diff
@@ -83,18 +83,26 @@
     {%- for fs in fsl %}
       {%- for src_file in src_files %}
         {%- if fs %}
-          {%- set fs_dir = salt['config.get'](fs, fs) %}
+          {%- set fs_dir_lookup = salt['config.get'](fs, fs) %}
         {%- else %}
-          {%- set fs_dir = salt['config.get'](tplroot ~ ':tofs:dirs:default', 'default') %}
+          {%- set fs_dir_lookup = salt['config.get'](tplroot ~ ':tofs:dirs:default', 'default') %}
         {%- endif %}
-        {%- set url = [
-            '- salt:/',
-            path_prefix_inc_ext.strip('/'),
-            files_dir.strip('/'),
-            fs_dir.strip('/'),
-            src_file.strip('/'),
-        ] | select | join('/') %}
+	{#- Force the `config.get` lookup result as a list #}
+	{%- if fs_dir_lookup is string %}
+	  {%- set fs_dirs = [fs_dir_lookup] %}
+	{%- else %}
+	  {%- set fs_dirs = fs_dir_lookup %}
+	{%- endif %}
+	{%- for fs_dir in fs_dirs %}
```

Comment:
We can lose the `else` and the extra variable doing it this way instead:

```diff
-	  {%- set fs_dirs = [fs_dir_lookup] %}
-	{%- else %}
-	  {%- set fs_dirs = fs_dir_lookup %}
+	  {%- set fs_dir_lookup = [fs_dir_lookup] %}
 	{%- endif %}
-	{%- for fs_dir in fs_dirs %}
+	{%- for fs_dir in fs_dir_lookup %}
```

Reply:
I apply what you suggested but used the `fs_dirs` for the lookup result.